### PR TITLE
Improved ITE() function

### PR DESCRIPTION
**Pull request by arihantparsoya**
Fix for #11530. Case when `b!=c` is added.
#### Sample Program:

``` python
>>> from sympy import *
>>> from sympy.abc import x
>>> ITE(x, False, True)
Not(x)
>>> ITE(x, True, False)
x
```

### CODE REVIEW

File: sympy/logic/boolalg.py [hunk 1]
```diff
@@ -929,6 +929,11 @@ def eval(cls, *args):
             return c
         if b == c:
             return b
+        else:
+            if b == True:
```
Comment (asmeurer) [inline sympy/logic/boolalg.py]: It's only if b == True and c == False.

Comment (parsoyaarihant) [inline sympy/logic/boolalg.py]: Thanks, changes are made.

**Comment by asmeurer** [comment]
Please add tests.

**Comment by parsoyaarihant** [comment]
* Tests are added.

**Comment by asmeurer** [comment]
In the future, make sure to write literally "fixes #11530". If you use any variation on that text it won't automatically close the issue.

**Pull request merged**